docs/guides/setup-dev.md

```diff
@@ -266,17 +275,6 @@ RUSTFLAGS as env var, or pass it in `config.toml` (either project level or globa
 rustflags = ["--cfg=no_cuda"]
 ```
 
-## Environment
-
-Edit the lines below and add them to your shell profile file (e.g. `~/.bash_profile`, `~/.zshrc`):
-
-```bash
-# Add path here:
-export ZKSYNC_HOME=/path/to/zksync
-
-export PATH=$ZKSYNC_HOME/bin:$PATH
-```
-
 ## Tips
 
 ### Tip: `mold`
```

Comment:
Make sure all tips are `h3`. The second one is an `h2`

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/3134/commits/d808f3dcd6a749700c3c0c5b5375a02d7ea1c827